ehrapy/plot/_survival_analysis.py

```diff
@@ -293,5 +296,132 @@ def kaplan_meier(
 
     if not show:
         return ax
+
     else:
         return None
+
+
+def coxph_forestplot(
+    coxph: CoxPHFitter,
```

Comment:
Could we actually redesign the interface and have this function take an AnnData object instead? We'd have to store the coxph results in the AnnData object. This would be much more in line with the rest of the ehrapy experience.

Also ping @eroell

Reply:
Yes, good point. Thanks. We should do this indeed - storing the results used for the plot in adata.uns I suppose

Reply:
Might also need a `key_added` argument then or something to ensure that we don't always overwrite results. But yeah this behavior must be consistent across all of the surv analysis methods

Reply:
We should be consistent about it and I think we should consider modifying all of them to take `AnnData`. It's what the rest of the API does

Reply:
> the kmf plot takes a list of KaplanMeierFitter Objects, so i went with that. in the survival analysis plots, ols is the only one taking an anndata.

Yes, noticed where the idea here came from. Having .tl writing to AnnData, and .pl to read from it should be our go to. It has not been done like that everywhere yet unfortunately, and we strive for this consistency now :)

Reply:
one way to go about this is to store the required table for plotting of the lifeline's fitter object in `adata.uns[key_added]` upon the `.tl` call;

`.uns` seems to me the only slot suitable for this table for the univariate functions such as kaplan_meier.

For cox_ph here, as the coefficients are on a per-variable level, I think adding them into the `.var` column would be kinda clean; basically adding into `.var` the columns `{key_added}_{"coef"}`, `{key_added}_{"coef lower 95%"}`, `{key_added}_{"coef upper 95%"}`. There could also be a boolean `{key_added}_{cox_ph_variable}` column, which the plotting function uses to pick only the variables used in the cox ph model